guides/mobile_app_guidance.md

```diff
@@ -0,0 +1,40 @@
+Mobile Application Guidance
+===========================
+
+Scope
+-----
+
+This guidance applies to mobile application in-house development and management.
+
+This guidance is intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+This document should be read in conjunction with the Mobile Application Standards.
```

Comment:
Suggest we link to those here if possible

Reply:
Done.